package.json

```diff
@@ -19,6 +19,7 @@
     "check-error.js"
   ],
   "main": "./index.js",
+  "version": "2.0.0-wip",
```

Comment:
We could change the version to something like `0.0.0-development` which is a very loud signal that its not a valid practical version, but it is still would hopefully solve the issues you're seeing.